File: C Programming/Labwork/ladderifelse.c
#include<stdio.h>

void main()
{
	int rno, s1, s2, s3, total;
	char name[5];
	float per;
	
	printf("\nEnter Roll No : ");
	scanf("%d" ,&rno);
	printf("\nEnter Name : ");
	scanf("%s" ,&name);
	printf("\nEnter Physics Marks : ");
	scanf("%d" ,&s1);
	printf("\nEnter Chemistry Marks : ");
	scanf("%d" ,&s2);
	printf("\nEnter Maths Marks : ");
	scanf("%d" ,&s3);
	total = s1+s2+s3;
	per = (float)total/3;
    printf("\nTotal P.C.M Marks : %d" ,total);
	printf("\nPercentage : %.2f" ,per);
	
	if(per >=75)
	{
		printf("\nGrade is : Distinction");
	}
	else if(per>=60)
	{
	    printf("\nGrade is : A");
	}
	else if(per>=50)
	{
		printf("nGrade is  : B");
	}
	else if(per>=40)
	{
		printf("\nGrade is : C");
	}
	else
	{
		printf("\nFail!");
	}
	
	
}

File: Jignesh_Python/Classwork/Ladderifelse.py
roll = int(input "Enter Roll No : "))
name = (input "Enter Name : "))
p = int(input "Enter Physics Marks : "))
m = int(input "Enter Maths Marks : "))
c = int(input "Enter Chemistry Marks : "))
total =(p+m+c)
per =(total)

print("*"*80)
print()
print("Roll No : ",roll)
print("Name : ",name)
print("Total : ",total)
print("Percentage : ",per)

if per>=75:
    print("Class is Distinction")
elif per>=60:
    print("Class is First Class")
elif per>=50:
    print("Class is Second Class")
elif per>=40:
    print("Class is Pass Class")
else:
    print("Fail!")
